src/widgets/panel/panel.tsx

```diff
@@ -248,7 +259,14 @@ const panel: PanelWidget = widgetParams => {
         return undefined;
       },
       render(...args) {
-        const [options] = args;
+        const [renderOptions] = args;
+
+        const options = {
+          ...(widget.getWidgetRenderState
+            ? widget.getWidgetRenderState(renderOptions)
+            : {}),
```

Comment:
Didn't this work?

```suggestion
          ...widget.getWidgetRenderState?.(renderOptions) || {},
```

Reply:
FYI https://github.com/algolia/instantsearch.js/pull/4527#discussion_r515244256

Reply:
All right next TS version upgrade will allow it then!